doc/classes/SceneTree.xml

```diff
@@ -238,76 +237,76 @@
 			<param index="0" name="node" type="Node" />
 			<description>
-				Emitted when a node's configuration changed. Only emitted in [code]tool[/code] mode.
+				Emitted when the [param node]'s [method Node.update_configuration_warnings] is called. Only emitted in the editor.
 			</description>
 		</signal>
 		<signal name="node_removed">
 			<param index="0" name="node" type="Node" />
 			<description>
-				Emitted whenever a node is removed from the [SceneTree].
+				Emitted when the [param node] exits this tree.
 			</description>
 		</signal>
 		<signal name="node_renamed">
 			<param index="0" name="node" type="Node" />
 			<description>
-				Emitted whenever a node is renamed.
+				Emitted when the [param node]'s [member Node.name] is changed.
 			</description>
 		</signal>
 		<signal name="physics_frame">
 			<description>
-				Emitted immediately before [method Node._physics_process] is called on every node in the [SceneTree].
+				Emitted immediately before [method Node._physics_process] is called on every node in this tree.
 			</description>
 		</signal>
 		<signal name="process_frame">
 			<description>
-				Emitted immediately before [method Node._process] is called on every node in the [SceneTree].
+				Emitted immediately before [method Node._process] is called on every node in this tree.
 			</description>
 		</signal>
 		<signal name="tree_changed">
 			<description>
-				Emitted whenever the [SceneTree] hierarchy changed (children being moved or renamed, etc.).
+				Emitted any time the tree's hierarchy changes (nodes being moved, renamed, etc).
 			</description>
 		</signal>
 		<signal name="tree_process_mode_changed">
 			<description>
-				This signal is only emitted in the editor, it allows the editor to update the visibility of disabled nodes. Emitted whenever any node's [member Node.process_mode] is changed.
+				Emitted when the [member Node.process_mode] of any node inside the tree is changed. Only emitted in the editor, to update the visibility of disabled nodes.
 			</description>
 		</signal>
 	</signals>
 	<constants>
 		<constant name="GROUP_CALL_DEFAULT" value="0" enum="GroupCallFlags">
-			Call a group with no flags (default).
+			Call nodes within a group with no special behavior (default).
 		</constant>
 		<constant name="GROUP_CALL_REVERSE" value="1" enum="GroupCallFlags">
-			Call a group in reverse scene order.
+			Call nodes within a group in reverse tree order.
 		</constant>
 		<constant name="GROUP_CALL_DEFERRED" value="2" enum="GroupCallFlags">
-			Call a group with a one-frame delay (idle frame, not physics).
+			Call nodes within a group after a one-frame delay (idle frame, not physics), similar to [method Object.call_deferred].
 		</constant>
 		<constant name="GROUP_CALL_UNIQUE" value="4" enum="GroupCallFlags">
-			Call a group only once even if the call is executed many times.
+			Call nodes within a group only once, even if the call is executed many times.
```

Comment:
Source: https://github.com/godotengine/godot/blob/471c42ee1fe2c84b4f248e7194a2b10b562ff508/scene/main/scene_tree.cpp#L234

Seems like deferred used to be the default for group calls, so that's maybe why this was not documented before.
```suggestion
			Call nodes within a group only once, even if the call is executed multiple times in the same frame. Must be combined with [constant GROUP_CALL_DEFERRED] to have any effect.
```

Reply:
The GROUP_CALL_DEFERRED requirement feels like a forgotten oddity that is worth keeping in mind.